docs/getting-started/features/tamper-detection.en.md

```diff
@@ -87,30 +73,30 @@ An attacker faces major challenges in replacing the firmware:
 
 - **Lack of Original Flash Data:** Without the exact original flash content, the attacker cannot reproduce the correct hash.
 
-- **Sequential Hash Dependency:** The hash function processes data sequentially (*TC Code*, UID, flash content), preventing the attacker from injecting or rearranging data to produce the same hash.
+- **Sequential Hash Dependency:** The hash function processes data sequentially (*TC Code*, device's UID, and flash memory contents), preventing the attacker from injecting or rearranging data to produce the same hash.
 
 - **One-Way Hash Functions:** Cryptographic hash functions like SHA-256 are one-way, making it infeasible to reverse-engineer or manipulate the hash without the original inputs.
 
 ### Why Tampered Firmware Cannot Bypass Verification
 
-- **Cannot Reconstruct the Hash:** Without the original flash data, the attacker cannot generate the correct hash, even if they know the UID and *TC Code* after the user enters it.
+- **Cannot Reconstruct the Hash:** Without the original flash data, the attacker cannot generate the correct hash, even if it knows the device's UID and the *TC Code* (after the user enters it).
```

Comment:
typo, referring to attacker as "it". Maybe "the attacker" could be replace by "attackers"

Reply:
or "they"